Dockerfile

```diff
@@ -172,6 +185,11 @@ RUN helm3 plugin install https://github.com/databus23/helm-diff.git --version v$
     && helm3 plugin install https://github.com/aslafy-z/helm-git.git --version ${HELM_GIT_VERSION} \
     && helm3 plugin install https://github.com/helm/helm-2to3 --version ${HELM_HELM_2TO3_VERSION}
 
+# Delete unneeded Helm plugin cache
```

Comment:
Fwiw, since this is a different `RUN` step, the previous layer will still contain the cache so it will not affect the final size of the image.

Reply:
I totally forgot about that. Let me fix.